Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptMergeController.java

```diff
@@ -0,0 +1,64 @@
+package edu.asu.conceptpower.web;
+
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+
+import edu.asu.conceptpower.app.bean.ConceptsMergeBean;
+import edu.asu.conceptpower.app.core.IConceptManager;
+import edu.asu.conceptpower.app.core.IConceptTypeManger;
+import edu.asu.conceptpower.app.core.POS;
+import edu.asu.conceptpower.app.exceptions.DictionaryDoesNotExistException;
+import edu.asu.conceptpower.app.exceptions.DictionaryModifyException;
+import edu.asu.conceptpower.app.exceptions.IndexerRunningException;
+import edu.asu.conceptpower.app.exceptions.LuceneException;
+import edu.asu.conceptpower.app.service.IConceptMergeService;
+import edu.asu.conceptpower.core.ConceptEntry;
+
+@Controller
+public class ConceptMergeController {
+
+    @Autowired
+    private IConceptManager conceptManager;
+
+    @Autowired
+    private IConceptMergeService conceptMergeService;
+
+    @Autowired
+    private IConceptTypeManger conceptTypesManager;
+
+    @RequestMapping(value = "auth/concepts/merge/prepare")
```

Comment:
We have one more request mapping auth/concepts/merge. Since this step just prepares the conceptmergebean object by obtaining the values from selected concept entries I had the requestmapping as auth/concepts/merge/prepare. When user edits all the values for conceptmergebean and click on merge auth/concepts/merge will be called.